TMessagesProj/jni/tgnet/ConnectionsManager.cpp

```diff
@@ -2523,14 +2522,12 @@ int32_t ConnectionsManager::getMtProtoVersion() {
 #ifdef ANDROID
 void ConnectionsManager::useJavaVM(JavaVM *vm, bool useJavaByteBuffers) {
     javaVm = vm;
-
-    JNIEnv *env = 0;
-    if (javaVm->GetEnv((void **) &env, JNI_VERSION_1_6) != JNI_OK) {
-        DEBUG_E("can't get jnienv");
-        exit(1);
-    }
-
     if (useJavaByteBuffers) {
+        JNIEnv *env = 0;
```

Comment:
this is the upstream way, it got tampered with when @slp was doing networkalarm I think, which he reverted leaving leftovers all over the place